addons/mail/static/src/widgets/form_renderer/form_renderer.js

```diff
@@ -155,8 +155,8 @@ FormRenderer.include({
      *
      * @override
      */
-    async _renderView() {
-        await this._super(...arguments);
+    async __renderView() {
```

Comment:
`_renderView` is overridden in a lot of different places, including `mail_enterprise`. I'm not sure why you needed to change this one, but you might need to change others based on the same reasoning.

Reply:
As long as the operations done in the override are sync, it's fine. The need for `__renderView` was to have a method to override to add async operations at render time.